packages/connection-form/src/components/form-help/form-help.tsx

```diff
@@ -29,13 +30,13 @@ const descriptionStyles = css({
 });
 
 const sectionDarkModeStyles = css({
-  '--theme-background-color': palette.blue.dark3,
-  '--theme-border-color': palette.blue.dark2,
+  '--theme-background-color': palette.gray.dark3,
```

Comment:
Note that according to the design, the blue help text boxes on the right are now grey.